### ISSUE TITLE
Online-Signature-Verification-using-Python

### ISSUE THREAD

**Issue by sdhnjhkj** [Arpx22/Online-Signature-Verification-using-Python#1]
Accuracy is 100% on a test set of 40 users, which is higher than the accuracy on the training set. How should this issue be addressed?
